README.md

```diff
@@ -159,6 +159,8 @@ mstorage.download("<media_id>"){result, error in
 ```
 
 ### List media ids
+* Without options
+
```

Comment:
You can add an explanation like "You'll get a default list if you set nothing or an empty Dictionary object on the first parameter." as in [media-storage-java](https://github.com/hishida/media-storage-java#list-media-ids).